src/cascadia/TerminalConnection/AzureConnection.cpp

```diff
@@ -1047,11 +1047,14 @@ namespace winrt::Microsoft::Terminal::TerminalConnection::implementation
     // and expects a numeric input corresponding to the shell of choice.
     std::optional<utility::string_t> AzureConnection::_ReadShellTypeFromUserInput()
     {
-        Shell shells[] = { { L"pwsh", L"Powershell" }, { L"bash", L"Bash" } };
+        const Shell shells[] = {
+            { L"pwsh", L"PowerShell" },
+            { L"bash", L"Bash" }
+        };
 
-        for (int i = 0; i < ARRAYSIZE(shells); i++)
+        for (size_t i = 0; i < ARRAYSIZE(shells); i++)
         {
-            _WriteStringWithNewline(_formatShell(i, shells[i]));
+            _WriteStringWithNewline(_formatShell(i, gsl::at(shells, i)));

```

Comment:
This is nuanced, but for places where we _know_ we're in-bounds we can use `til::at`. `gsl::at` does a bounds check :smile: which we don't need here.

Reply:
Thanks! :) I was looking for it 😊 Fixing it.